src/CMakeLists.txt

```diff
@@ -186,6 +186,8 @@ list( APPEND SOURCES
       MixAndRender.h
       MixerBoard.cpp
       MixerBoard.h
+      MovableControl.cpp
+      MovableControl.h
```

Comment:
And this file.

But whatever the folders they are in,  it is good to see RealtimeEffectPanel shrink to its essentials.